src/common/src/main/scala/osmesa/common/functions/osm/package.scala

```diff
@@ -224,6 +224,10 @@ package object osm {
     (_: Map[String, String]).contains("highway")
   }
 
+  val isCoastline: UserDefinedFunction = udf {
+    (_: Map[String, String]).getOrElse("natural", "no").toLowerCase == "coastline"
```

Comment:
Seriously tiny nit: I would default to `""` rather than `"no"` here. It doesn't change anything, but reading through, it made me wonder, "why would it be 'no'?"